cli/util.go

```diff
@@ -15,18 +15,22 @@ type apiPort struct {
 	Port   string
 }
 
-type Volumes struct {
-	DefaultVolumes      []string
-	NodeSpecificVolumes map[string][]string
-}
-
 const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 const (
 	letterIdxBits = 6                    // 6 bits to represent a letter index
 	letterIdxMask = 1<<letterIdxBits - 1 // All 1-bits, as many as letterIdxBits
 	letterIdxMax  = 63 / letterIdxBits   // # of letter indices fitting in 63 bits
 )
 
+// defaultNodes describes the default node group (master)
```

Comment:
Maybe we should have a `types.go` for those things, but that's not part of this PR :+1: